Voycar.Api.Web/Features/Members/Post/Registration/Request.cs

```diff
@@ -0,0 +1,25 @@
+namespace Voycar.Api.Web.Features.Members.Post.Registration;
+
+using System.ComponentModel.DataAnnotations;
+
+public class Request
+{
+    // Login
+    [EmailAddress]
+    public required string Email{ get; set; }
+    public required string Password { get; set; }
+    public required string ConfirmPassword { get; set; }
```

Comment:
Da wir die Passwortüberprüfung (enthaltene Zeichen) auf das Frontend verlegen, würde ich diese Überprüfung auch nur im Frontend machen. (hier also weg) Entsprechend muss dann `Registration.Endpoint.cs` in Zeile 34 angepasst werden und der Kommentar in Zeile 9.

Reply:
seh ich auch so. Also `ConfirmPassword` einfach weg, wir checken im Frontend, ob man zweimal das richtige Passwort eingibt, und schicken es dann nur einmal ans Backend.